src/pages/patientView/PatientViewPage.tsx

```diff
@@ -167,6 +172,37 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         });
     }
 
+    fetchCosmicData(mutations:Mutation[]):Promise<ICosmicData> {
+        const generateMap = function(cosmicMutations:CosmicMutation[]) {
```

Comment:
In addition to that, we should also consider externalizing data fetcher functions to other classes